numpy/core/src/multiarray/compiled_base.c

```diff
@@ -795,7 +795,52 @@ static int sequence_to_arrays(PyObject *seq,
     return 0;
 }
 
-/* Inner loop for unravel_index */
+/*
+ * Force-cast an array into intp if it is empty. Fail otherwise.
+ *
+ * op        - Pointer to the array pointer. Will be modified if
+ *             recasting is needed.
+ * paramname - Name to be used for error string.
+ */
+static int _cast_to_intp_or_fail(PyArrayObject **op,
```

Comment:
I know it is not always respected, `static int` gets its own line, take a look at [our C style guide](https://github.com/numpy/numpy/blob/master/doc/C_STYLE_GUIDE.rst.txt#code-lay-out).